third_party/ObjexxFCL/src/ObjexxFCL/vectorize.hh

```diff
@@ -23,16 +23,33 @@
 #define ASSUME(b)
 #define ASSUME_ALIGNED(p,b) p=(decltype(p))__builtin_assume_aligned(p,b)
 #define RESTRICT __restrict__
-#elif defined(_WIN32) && ( defined(_MSC_VER) || defined(__INTEL_COMPILER) )
+
+#elif defined(_WIN64) && ( defined(_MSC_VER) || defined(__INTEL_COMPILER) )
 #define ASSUME(b) __assume(b)
 #define ASSUME_ALIGNED(p,b) __assume_aligned(p,b)
 #define RESTRICT __restrict
+
+// Disable alignment hints for 32-bit windows builds
+#elif defined(_WIN32) && ( defined(_MSC_VER) || defined(__INTEL_COMPILER) )
+#define ASSUME(b) __assume(b)
+#define ASSUME_ALIGNED(p,b) 
+#define RESTRICT __restrict
+
 #else
 #define ASSUME(b)
 #define ASSUME_ALIGNED(p,b)
 #define RESTRICT
 #endif
 
+// Alignment for Compiler Options
+#if defined (__AVX2__)
```

Comment:
@DeadParrot The stuff I moved into ObjexxFCL/vectorize.hh had nothing to do with EnergyPlus. I did leave some stuff in EnergyPlus/Vectorize.hh (and added some stuff too) that had to do with the size of Real64 and the resulting vector lengths and alignment macros. (If I knew how to highlight diffs in git I would do it here). If you explicitly don't want that stuff in ObjexxFCL because it has nothing to do with it either, I can move it back to the EnergyPlus side.